.vortex/installer/tests/Functional/Command/BuildCommandTest.php

```
@@ -53,14 +53,14 @@ public function testBuildCommand(
 
     // Mock getOutput() to handle both string and array returns.
     // For 'docker' command (compose config), return JSON with
-    // VORTEX_LOCALDEV_URL. For other commands, return standard mock output.
+    // LOCALDEV_URL. For other commands, return standard mock output.
     $build_process_runner->method('getOutput')->willReturnCallback(function (bool $as_array = FALSE) use (&$current_command, $docker_compose_url): array|string {
```

PHPMD (2.15.0)

[error] The method testBuildCommand has a boolean flag argument $as_array, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)